servers/zms/src/main/java/com/yahoo/athenz/zms/DBService.java

```diff
@@ -1855,7 +1855,7 @@ Role getRole(ObjectStoreConnection con, String domainName, String roleName,
                 // if we have no trust field specified then we need to
                 // retrieve our standard group role members
                 
-                role.setRoleMembers(con.listRoleMembers(domainName, roleName));
+                role.setRoleMembers(con.listRoleMembers(domainName, roleName, pending));
```

Comment:
after this call, for backward support we set the members list as well (which is just a list of member names). when pending value is set to true we can't include the members in the list since that gives the wrong impression so we should always exclude the pending members from the convertRoleMembersToMembers method.